components/batch-system/src/config.rs

```diff
@@ -13,8 +13,15 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Config {
+        // FIXME: if hibernate regions is enabled by default on release branch, we can use
+        // the same configuration here.
+        let max_batch_size = if tikv_util::build_on_master_branch() {
```

Comment:
How about changing it at `check` method according to the value of `hibernate-region`?

Reply:
What if users set the `max-batch-size`?  Use a `Option`?

Reply:
Yes.